packages/cli/src/core/types.flow.js

```diff
@@ -18,7 +18,7 @@ export type LocalCommandT = {
     command: string,
     description?: string,
     parse?: (val: string) => any,
-    default?: ((ctx: ContextT) => mixed) | mixed,
```

Comment:
turned out the only `default` values we use are `string | boolean | number `

Reply:
Sweet 🍰